Trouble with new build

I'm trying to build the new version of dcurl and I am getting the following error after "make clean".  

tyler@arm:~/dcurl-update/dcurl$ make BOARD=de10nano BUILD_FPGA_ACCEL=1 BUILD_COMPAT=1
readlink: missing operand
Try 'readlink --help' for more information.
  CC    build/compat_ccurl.o
src/compat_ccurl.c: In function ‘ccurl_pow’:
src/compat_ccurl.c:23:9: error: too few arguments to function ‘dcurl_init’
         dcurl_init();
         ^~~~~~~~~~
In file included from src/compat_ccurl.c:11:0:
src/dcurl.h:51:6: note: declared here
 bool dcurl_init(dcurl_config *config);
      ^~~~~~~~~~
Makefile:176: recipe for target 'build/compat_ccurl.o' failed
make: *** [build/compat_ccurl.o] Error 1

--- a/src/compat_ccurl.c
+++ b/src/compat_ccurl.c
@@ -20,7 +20,7 @@ char *ccurl_pow(char *trytes, int mwm)
 {
     pthread_mutex_lock(&mtx);
     if (!is_initialized) {
-        dcurl_init();
+        dcurl_init(NULL);
         is_initialized = true;
     }
     pthread_mutex_unlock(&mtx);

